misk/src/main/kotlin/misk/web/marshal/Generic.kt

```diff
@@ -0,0 +1,111 @@
+package misk.web.marshal
+
+import misk.inject.typeLiteral
+import misk.web.Response
+import misk.web.ResponseBody
+import misk.web.marshal.Marshaller.Companion.actualResponseType
+import okhttp3.MediaType
+import okio.BufferedSink
+import okio.BufferedSource
+import okio.ByteString
+import java.lang.reflect.ParameterizedType
+import java.lang.reflect.Type
+import kotlin.reflect.KParameter
+import kotlin.reflect.KType
+import kotlin.reflect.jvm.javaType
+
+/** Handles generic unmarshalling, for cases where the action can accept anything */
+object GenericUnmarshallers {
+    fun canHandle(type: KType) = canHandle(type.javaType)
+    fun canHandle(type: Type) = GENERIC_REQUEST_TYPES.contains(type)
+
+    private val GENERIC_REQUEST_TYPES = setOf(
+            String::class.java,
+            Array<Byte>::class.java,
```

Comment:
I wonder if we should deliberately skip this, since it doesn’t do more than ByteString and ByteString is good

Reply:
I'm cool with that